src/containers/MovieList.container.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux'
+import MovieList from '../components/MovieList';
+
+const mapStateToProps = state => ({
```

Comment:
You could use destruction here to write this code shorter:
`const mapStateToProps = ({ movies: { movieList, sortingType })=> ({ movieList, sortingType });`

Reply:
done